### PR TITLE
Feature/semantic exercise

### PR DESCRIPTION
**Pull request by OscarBosigas**

### CODE REVIEW

File: 3-semantic-html/semantic.html [hunk 1]
```diff
@@ -10,26 +10,40 @@
...
+    
     <main>
-        <!-- Content that is unique to a single web page -->
+
+        <article>
+            <header>
+                <h1></h1>
+                <content></content>
+            </header>
+
```
Comment (juancho11gm) [inline 3-semantic-html/semantic.html]: Please remove all the empty lines of your file

**Review by juancho11gm — CHANGES_REQUESTED** [review]
You have the structure of the HTML. Now you have to fill the elements with content. For example, you can use the `<li>` tags in the `<nav`> with items that you want to mention on the web page. The same with all the other elements that are empty.

File: 3-semantic-html/semantic.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<body>
+    <header>
+        <nav>
+        </nav>
+    </header>
+    <main>
+        <article>
+            <header>
+                <h1></h1>
```
Comment (pushdev-code) [inline 3-semantic-html/semantic.html]: Please include some text so it's not just empty.

File: 3-semantic-html/semantic.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                <content></content>
+            </header>
+            <section>
+                <p></p>
+                <content></content>
```
Comment (pushdev-code) [inline 3-semantic-html/semantic.html]:
```suggestion
```
The content should be inside `<p></p>` from line 23

**Review by pushdev-code — CHANGES_REQUESTED** [review]
Please include texts inside the tags, and remove the empty `<content>` tags

**Pull request merged**